scripts/eastern_kingdoms/blackrock_depths/blackrock_depths.h

```diff
@@ -87,8 +96,20 @@ enum
     SPELL_STONED            = 10255,                        // Aura of Warbringer Constructs in Relict Vault
 
     FACTION_DWARF_HOSTILE   = 754,                          // Hostile faction for the Tomb of the Seven dwarfs
+    FACTION_ARENA_NEUTRAL   = 674,                          // Neutral faction for NPC in top of Arena after event complete
```

Comment:
@Tobschinski provided me data a few minutes ago saying that faction should instead be 15 and not 674.
```
[25] UpdateType: Values
[25] Object Guid: Full: 0x1C16FC1CC008B000003A08000308899B Creature/0 R1471/S14856 Map: 230 Entry: 8896 Low: 50891163
[25] UNIT_FIELD_FACTIONTEMPLATE: 15
```